src/View/Helper/MigrationHelper.php

```diff
@@ -0,0 +1,250 @@
+        }
+
+        return $columns;
+    }
+
+
+    /**
+     * Returns an array of column data for a single column
+     *
+     * @param Cake\Database\Schema\Table $tableSchema Name of the table to retrieve columns for
+     * @param string $column A column to retrieve data for
+     * @return array
+     */
+    public function column($tableSchema, $column)
+    {
+        return [
+            'columnType' => $tableSchema->columnType($column),
+            'options' => $this->attributes($tableSchema->name(), $column),
+        ];
+    }
+
+    /**
+     * Returns a string-like representation of a value
+     *
+     * @param string $value A value to represent as a string
+     * @return mixed
+     */
+    public function value($value)
+    {
+        if ($value === null || $value === 'null' || $value === 'NULL') {
+            return 'null';
+        }
+
+        if ($value === 'true' || $value === 'false') {
+            return $value;
+        }
+
+        if (is_bool($value)) {
+            return $value ? 'true' : 'false';
+        }
+
+        if (is_numeric($value) || ctype_digit($value)) {
+            return (int)$value;
+        }
+
+        return sprintf("'%s'", addslashes($value));
+    }
+
+    /**
+     * Returns an array of attributes for a given table column
+     *
+     * @param string $table Name of the table to retrieve columns for
+     * @param string $column A column to retrieve attributes for
+     * @return array
+     */
+    public function attributes($table, $column)
+    {
+        $collection = $this->config('collection');
+        $tableSchema = $collection->describe($table);
+        $validOptions = [
+            'length', 'limit',
+            'default', 'null',
+            'precision', 'scale',
+            'after', 'update',
+            'comment', 'unsigned',
+            'signed', 'properties'
+        ];
+
+        $attributes = [];
+        $options = $tableSchema->column($column);
+        foreach ($options as $_option => $value) {
+            $option = $_option;
+            switch ($_option) {
+                case 'length':
+                    $option = 'limit';
+                    break;
+                case 'unsigned':
+                    $option = 'signed';
+                    $value = (bool)!$value;
+                    break;
+                case 'unique':
+                    $value = (bool)$value;
+                    break;
+            }
+
+            if (!in_array($option, $validOptions)) {
+                continue;
+            }
+
+            $attributes[$option] = $value;
+        }
+
+        ksort($attributes);
+        return $attributes;
+    }
+
+    /**
+     * Returns an array converted into a formatted multiline string
+     *
+     * @param array $list array of items to be stringified
+     * @param array $options options to use
+     * @return string
+     */
+    public function stringifyList(array $list, array $options = [])
```

Comment:
Why not fix bake's helper? If you can give me/explain a failing use case I can take care of that.